docs: Added guide for the error which occurs during installation of Kubearmor (#1511)
accuknox/KubeArmor: getting-started/FAQ.md

> docker exec -it kind-control-plane bash -c "apt update && apt install apparmor-utils -y && systemctl restart containerd"

Reviewer: Please update this FAQ to generic way as well. There are cases where AppArmor is availble but not enabled. We need to instruct them to install apparmor utils.

Example CBL Mariner. You can see the updater script we install apparmor utils there as well.

So the goal is to update the FAQ in a generic manner such that people can understand why do they need to enable AppArmor utils, how to check whether apparmor availability.

Why is the additional logic is needed for dockerized nodes.

```

> The above command will execute the bash script to install the apparmor in the kind-control-plane, we can use the similar command to install apparmor in minikube as well and all the other docker based kubernetes tools.

Reviewer 2: ```suggestion
The above command will install the AppArmor utilities in the kind-control-plane, we can also use this command to install these in minikube as well as in all the other docker based Kubernetes environments.
```

docker exec -it kind-control-plane bash -c "apt update && apt install apparmor-utils -y && systemctl restart containerd"
```

The above command will install the AppArmor utilities in the kind-control-plane, we can also use this command to install these in minikube as well as in all the other docker based Kubernetes environments.

After this, exit out of the node shell and follow the [getting-started guide](https://github.com/kubearmor/KubeArmor/blob/main/getting-started/deployment_guide.md).

It might be possible that apart from the dockerized kubenetes environment AppArmor might not be available on the master node itself in the Kubernetes cluster. To check for the same you can run the below command to check for the AppArmor support in kernel config:

```
cat /boot/config-$(uname -r) | grep -e "APPARMOR"
```

Following flags need to exist and set to `y`
```ini
CONFIG_SECURITY_APPARMOR=y
CONFIG_SECURITY_APPARMOR_INTROSPECT_POLICY=y
CONFIG_SECURITY_APPARMOR_HASH=y
CONFIG_SECURITY_APPARMOR_HASH_DEFAULT=y
CONFIG_SECURITY_APPARMOR_EXPORT_BINARY=y

> CONFIG_SECURITY_APPARMOR_PARANOID_LOAD=y

Reviewer: ```suggestion
CONFIG_SECURITY_APPARMOR=y
```

We only need this flag.

```

Run the command to install apparmor:

```
apt update && apt install apparmor-utils -y
```

> You need to restart your master node in-order to make APPARMOR available as a kernel config security.

Reviewer: Restarting of node is not needed with apparmor utils. On the restart of CRI is needed.
